mylist.rb

```diff
@@ -0,0 +1,25 @@
+require_relative './my_enumerable'
+
+class MyList
+  include MyEnumerable
+
+  def initialize(list)
```

Comment:
- [ ] When I try to add multiple numbers as asked for testing here I get an error because the code expects only one argument, instead please make this as following in order to be able to expect many elements:
    def initialize(*args)

![image](https://user-images.githubusercontent.com/62705947/223856141-2d8a3d12-236a-4e6a-8b48-8cdefa075cf3.png)
![image](https://user-images.githubusercontent.com/62705947/223856533-6e429613-61e2-44be-b27e-a88c172bf152.png)

Reply:
![Captura de pantalla (390)](https://user-images.githubusercontent.com/79432268/223860272-b10c28c6-cbaa-4dd0-85d4-c5ea3571a90d.png)

Sorry but I do not understand. When executing I get the results you request.

Reply:
You are requesting like this list = Mylist.new([1, 2, 3, 4)] **(as one array),** while on the requirements testing it is like Mylist.new(1, 2, 3,4). On your example you are sending one array meaning one element, while on the requirements it's asking to be able to send as many elements as you want.

Reply:
great! and thank you very much for your review. I perfectly understood what he asked me

Reply:
I am glad I could help